ui/page/dcrdex/market.go

```diff
@@ -0,0 +1,1014 @@
+								Width:       cryptomaterial.MatchParent,
+								Height:      cryptomaterial.WrapContent,
+								Margin:      layout.Inset{Bottom: u10, Top: u10},
+								Orientation: layout.Horizontal,
+							}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									return pg.immediateOrder.Layout(gtx)
+								}),
+								layout.Rigid(func(gtx C) D {
+									return cryptomaterial.LinearLayout{
+										Width:     cryptomaterial.WrapContent,
+										Height:    cryptomaterial.WrapContent,
+										Clickable: pg.immediateMoreInfo,
+										Padding:   layout.Inset{Top: u10, Left: u2},
+									}.Layout2(gtx, pg.Theme.Icons.InfoAction.Layout16dp)
+								}),
+							)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+								layout.Flexed(1, pg.createOrderBtn.Layout),
+							)
+						}),
+					)
+				}),
+				layout.Stacked(func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							return pg.toggleBuyAndSellBtn.Layout(gtx)
+						}),
+						layout.Flexed(1, func(gtx C) D {
+							return layout.Inset{Bottom: u10, Top: u10}.Layout(gtx, func(gtx C) D {
+								return pg.orderTypesDropdown.Layout(gtx, 0, true)
+							})
+						}),
+					)
+				}),
+			)
+		})
+	}
+}
+
+func (pg *DEXMarketPage) orderbook() func(gtx C) D {
+	// TODO: Use real values
+	makeOrderBookBuyOrSell := func(sell bool) []layout.FlexChild {
+		var mockOrderBook []layout.FlexChild
+		for i := 0; i < maxOrderDisplayedInOrderBook; i++ {
+			ord := &order{
+				isSell: sell,
+				price:  0.001268,
+				amount: 1.003422,
+				epoch:  34534566,
+			}
+
+			mockOrderBook = append(mockOrderBook, layout.Rigid(pg.orderBookRow(
+				textBuyOrSell(pg.Theme, ord.isSell, pg.Printer.Sprintf("%f", ord.price)),
+				pg.Theme.Body2(pg.Printer.Sprintf("%f", ord.amount)),
+				pg.Theme.Body2(fmt.Sprintf("%d", ord.epoch)),
+			)))
+		}
+		return mockOrderBook
+	}
+
+	return func(gtx C) D {
+		return cryptomaterial.LinearLayout{
+			Width:       gtx.Dp(orderFormAndOrderBookWidth),
+			Height:      gtx.Dp(orderFormAndOrderBookHeight),
+			Background:  pg.Theme.Color.Surface,
+			Margin:      layout.Inset{Top: u5, Bottom: u5},
+			Padding:     layout.UniformInset(u16),
+			Border:      cryptomaterial.Border{Radius: cryptomaterial.Radius(8)},
+			Orientation: layout.Vertical,
+			Direction:   layout.Center,
+		}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return pg.semiBoldLabelText(gtx, values.String(values.StrOrderBooks))
+					}),
+					layout.Flexed(1, func(gtx C) D {
+						return layout.E.Layout(gtx, pg.seeFullOrderBookBtn.Layout)
+					}),
+				)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{Top: u10}.Layout(gtx, pg.orderBookRow(
+					semiBoldGray3Size14(pg.Theme, values.StringF(values.StrAssetPrice, "BTC")),
```

Comment:
BTC text should already be localized we can use that or we can get it from the selected asset, `asset.GetAssetType().String()`

```suggestion
					semiBoldGray3Size14(pg.Theme, values.StringF(values.StrAssetPrice, "BTC")),
```

Reply:
> BTC text should already be localized 

Okay.

Reply:
> BTC text should already be localized we can use that 

None for BTC but there's an English entry for only DCR.